### ISSUE TITLE
adapt.min.js does not get uglified

### ISSUE THREAD

**Issue by NayanKhedkar** [adaptlearning/adapt_framework#3350]
### Subject of the issue/enhancement/features
on  execution of grunt build the adapt.min.js file code looks readable it's not uglifying properly.

### Your environment
* version (AT/Framework)
* which browser and its version
* device(s) + operating system(s)

### Steps to reproduce
execute the command grunt build 
### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Screenshots (if you can)
![image](https://user-images.githubusercontent.com/16237837/207289713-7d08a760-1c32-44e2-83fc-4c8157977b14.png)

**Comment by oliverfoster** [comment]
These lines should be minifying:
https://github.com/adaptlearning/adapt_framework/blob/1b8607b7fe332f143c19a8541cde1651dc7832ed/grunt/tasks/javascript.js#L374-L378

It does on my kit:

![image](https://user-images.githubusercontent.com/7974663/207807594-283d2b58-5793-4fbf-98f3-7c470db964cb.png)
FW: 5.24.4

**Comment by oliverfoster** [comment]
You have formatted the code in chrome.

**Comment by oliverfoster** [comment]
Adapt APIs should be accessible to external code. Name mangling would prevent that.
All of the files in `libraries/` folders aren't compiled, they rely on the unmangled names in some circumstances.

"Code is easily readable" why is this a problem, except for the larger output size? It helps debugging edge cases when sourcemaps aren't on.

**Comment by NayanKhedkar** [comment]
True, but if anyone doesn't want to easily expose component/extension code to outside world 😄

**Comment by oliverfoster** [comment]
All in-browser, publically accessible JavaScript is reverse engineerable. It can be obfuscated, but it's still relatively easy to reverse as the code comes in plain text and is visible with in-browser debugging tools.
JavaScript tends to reference unmangled browser APIs (setTimeout, createElement, etc) and often needs to expose itself to external code as an API, mangling for obfuscation is really limited in its usefulness and comes with a lot of downsides.

Mangling is quite useful for reducing output size, by about 10%. https://lihautan.com/reduce-minified-code-size-by-property-mangling/

Mangling is not a good solution for obfuscation on its own and even heavily obfuscated javascript can be reverse engineered with a little effort. https://www.javascriptobfuscator.net/

**Comment by NayanKhedkar** [comment]
make sense 👍
